tests/foreman/ui/test_settings.py

```diff
@@ -353,9 +353,44 @@ def test_positive_update_email_delivery_method_sendmail():
     :CaseImportance: Critical
 
     :CaseLevel: Acceptance
-
-    :CaseAutomation: notautomated
     """
+    mail_config_default_param = {
+        "delivery_method": "",
+        "email_reply_address": "",
+        "email_subject_prefix": "",
+        "sendmail_arguments": "",
+        "sendmail_location": "",
+        "send_welcome_email": "",
+    }
+    for content in mail_config_default_param:
+        mail_config_default_param[content] = [
+            ele.value
+            for ele in entities.Setting().search(query={'search': 'name={}'.format(content)})
+        ][0]
+
+    mail_config_new_params = {
+        "delivery_method": "Sendmail",
+        "email_reply_address": gen_email(),
+        "email_subject_prefix": [gen_string('alpha')],
+        "sendmail_location": "/usr/sbin/sendmail",
+        "send_welcome_email": "Yes",
+    }
+
+    with session:
+        try:
+            for mail_content in mail_config_new_params:
+                session.settings.update(
+                    "{}".format(mail_content), "{}".format(mail_config_new_params[mail_content])
+                )
```

Comment:
I presume there is no API/CLI way to send this email, so we need UI for that. But do we have to set all options using UI? I believe it would be much faster if we used API instead, and particular way we set config is probably not important overall.

Reply:
The intent behind this to check the email contents are updated or not through UI, Even though we can achieve the same from CLI. The complete update operation from UI would take 9 to 10 seconds(sometimes we can not quantify the automation performance over quality). And this is the only test case in UI(Settings, sendmail/smtp) where we are doing email related update, I still have some test cases related to Sendemail/smtp, I will prefer to use API there.

Reply:
Personally I don't see much value in adding coverage for different settings, but if it takes only ~10 seconds, then there's also no reason to not use UI.